assets/css/card.css

```diff
@@ -45,6 +45,53 @@
   top: 4px;
   left: 12px;
 }
+/******************************************* Testimonials card *********************************/
+.testimonial-card {
+  background: #f1f3f2;
+  border-radius: 18px;
+  height: 371px;
+  position: relative;
```

Comment:
Do you really need it?

Reply:
Can you describe problem? The question is "why do i need this class?" ?

Reply:
position: relative;